MonoGame.Framework/Graphics/Effect/EffectMaterial.cs

```diff
@@ -1,7 +1,7 @@
 #region License
 /*
 Microsoft Public License (Ms-PL)
-MonoGame - Copyright © 2009 The MonoGame Team
+MonoGame - Copyright © 2009 MonoGame Foundation, Inc
```

Comment:
Maybe fix this to use the normal header now?

Also remove the #region since we don't do that anywhere else.